src/common.ts

```diff
@@ -34,3 +35,16 @@ export function getLastTimestamp(): number {
 	const file = JSON.parse(fs.readFileSync(fileToCheck).toString());
 	return file.timestamp;
 }
+
+export function tiebreakerSortFactory(timeframe: 'weekly' | 'total', walletsStats: WalletsStats) {
```

Comment:
Could be helpful to add a quick note that this will sort wallets first by highest byte count, if those are the same it will sort by highest community tips, otherwise it will sort by the wallet that has been participating in inferno the longest

Thanks for breaking this function out here!

Reply:
bbcbc77dc8a3c2583c7bf8ca18b1ff97bff95abc